memote/support/basic.py

```diff
@@ -51,7 +51,8 @@ def find_nonzero_constrained_reactions(model):
     lower_bound, upper_bound = helpers.find_bounds(model)
     return [rxn for rxn in model.reactions if
             0 > rxn.lower_bound > lower_bound or
-            0 < rxn.upper_bound < upper_bound]
+            0 < rxn.upper_bound < upper_bound or
+            (rxn.lower_bound == 0 and rxn.upper_bound < upper_bound)]
```

Comment:
It'd be somewhat strange but the bounds could be set such that the reaction can only run in the reverse direction.

Reply:
Basically the reverse of the last condition:
```python
(rxn.upper_bound == 0 and rxn.lower_bound > lower_bound)
```
but looking at it again, I now see that these two cases are actually already covered by the first two conditions since they are connected by OR. The last condition thus is superfluous.